### DISCUSSION TITLE
Where is the script to generate Tensorrt format models ?

### DISCUSSION THREAD

**Discussion by philippedegeer** [blakeblackshear/frigate#13394] [Ask A Question] [answered]
### What is your question?

Hello all,
I have been able to setup Tensorrt models leveraging my NVIDIA GPU using the provided models. It is working very well.
I would like now to test other models. 
The documentations says "A script is included that will build several common models". I have searched in the docker container files but could not locate this script.
Where is this script located in the docker file ? I would like to see it to understand more about the trt creation process. 
Thanks
Philippe

**Comment by NickM-27** [comment, marked as answer]
https://github.com/blakeblackshear/frigate/blob/dev/docker/tensorrt/detector/rootfs/etc/s6-overlay/s6-rc.d/trt-model-prepare/run

**Reply by philippedegeer** [reply]
Many thanks Nick !
